### ISSUE TITLE
preview

### ISSUE THREAD

**Issue by PietroLiuzzo** [SChAth/ScAthiop#67]
perfetto, una domanda: sotto additions io vedo solo quel che è nel `<desc>` ma non nel `<q>` o `<note>` o `<date>` . Forse i `<q>` vanno all’interno del `<desc>`? Sarebbero gli inizi ma `<incipit>` non ci posso usare  mi sa...

**Comment by PietroLiuzzo** [comment]
@eu-genia the preview currently shows you the content of `locus`, `q` and `desc`. 

I can add `note` and `date` when it occurs in `desc` or `note`, and update the guidelines with an example as soon as there is one.

I have some observations on the following examples:

````
<item xml:id="a21" xml:lang="gez" corresp="LIT2663Mahlet">
   <locus target="#154r"/>
   <note> <ref corresp="#h7">hand as before</ref> </note>
</item>

<item xml:id="a21" xml:lang="gez">
   <locus target="#154r"/>
   <desc type="Poem" corresp="LIT2663Mahlet"/>
   <note> <ref corresp="#h7">hand as before</ref> </note>
</item>
````

`@xml:lang` should be used for `q` not for the item, according to current guidelines. the content of `item` is not geez. 

Also, a desc without content is not very useful IMHO. You could write the title of this addition in desc and mark it up, assigning to title the `@corresp` for example.

````
<desc type="Poem"><title corresp="LIT2663Mahlet">poem</title></desc>
                            
````
Also, for internal linking in `ref` we opted for `@target` and not `@corresp`.
https://github.com/SChAth/ScAthiop/wiki/guidelines#references 

but this is a good point to fix in the guidelines and schema as the text in the guilines is perhaps too short and the schema message for target and corresp deceiving. I will update that immediately, as well as the handling of ref which is to basic.

**Comment by eu-genia** [comment]
io volevo distinguire fra additioni in geez e quelle in amarico, non ho citazioni, solo l’informazione sulla lingua, come faccio?

**Comment by MassimoVilla** [comment]
In ms. BLadd16187 ho dato l'informazione nel seguente modo: ` <q xml:lang="am"/>`, con un `<q>` privo di contenuto. In quel caso si trattava di "Varia" (e2), ma immagino sia lo stesso problema.

**Comment by PietroLiuzzo** [comment]
we where discussing whether to temporarily put it in `desc` but I actually like in this case what you propose @MassimoVilla. if no one has anything to argue against this I'll stick it in the guidelines

**Comment by eu-genia** [comment]
`<origDate notBefore="1900" notAfter="1960">King Täklä Giyorgis (reigned for several periods between 1779 and 1800) is mentioned twice on fol. 30r. However, the Ms. is definitly recent.</origDate>
`
corrispondeva a
"Dating :	from  1900   to  1960
(comment) :	King Täklä Giyorgis (reigned for several periods between 1779 and 1800) is mentioned twice on fol. 30r. However, the Ms. is definitly recent."

nella trasformazione è visibile solo il comment, non dating

**Comment by PietroLiuzzo** [comment]
That is now fixed. Please wait until I upload the new version of preview
transform
Il 10/Mag/2016 10:00, "eu-genia" <notifications@github.com> ha scritto:

**Comment by eu-genia** [comment]
`<condition key="intact"/>`
corresponding to
State of preservation :	intact
shows
nothing in preview

**Comment by eu-genia** [comment]
can we have height:width proportions in visualization? (as we had in domlib)

**Comment by eu-genia** [comment]
```
                  <bindingDesc>
                     <binding>
                        <decoNote xml:id="b1">Two wooden boards covered with reddish-brown tooled leather; textile inlays.&#xD;
Two pairs of sewing stations.</decoNote>
                        <decoNote xml:id="b2" type="bindingMaterial">
                           <material key="wood"/>
                           <material key="leather"/>
                           <material key="textile"/>
                        </decoNote>
                        <decoNote xml:id="b3" type="Endbands"/>
                        <decoNote xml:id="b4" type="Other">Two pairs of crude endband stitches at the spine cover.</decoNote>
                     </binding>
                  </bindingDesc>
```

decoNote b1 is visualized outside Binding (directly under Physical description)

**Comment by eu-genia** [comment]
and decoNote b4 appears under Decoration instead of Binding

**Comment by eu-genia** [comment]
```
                           <dimensions type="margin">
                              <dim type="top">17</dim>
                              <dim type="bottom">36</dim>
                              <dim type="right">18</dim>
                              <dim type="left">13</dim>
                              <dim type="intercolumn">10</dim>
                           </dimensions>
                           <note>All data for fol. 5r.</note>
```

the note is not shown

**Comment by eu-genia** [comment]
`<layout columns="2" writtenLines="17">
`not shown

**Comment by eu-genia** [comment]
abbreviations are shown but the subtitle "abbreviations" is missing would be helpful

**Comment by eu-genia** [comment]
in the domlib generated mss the biblio part e.g.
```
                     <listBibl type="mss">
                        <bibl>EMML no. 3012 (18th cent., church of St. Mikaʾel, Ḥarr ʾAmba, Shoa), fols. 10r-20v (EMML VIII, 18-19)</bibl>
                        <bibl>EMML no. 31 (20th cent. (?), church of St. Mark, ʾƎnṭoṭṭo), fols. 159r-192v (EMML I, 32);</bibl>
                        <bibl>EMML no. 48 (20th cent., church of St. Raguʾel, ʾƎnṭoṭṭo), fols. 75r-99r (EMML I, 53);</bibl>
                        <bibl>EMML no. 502 (20th cent., church of St. Täklä Haymanot, quarter of Täklä Haymanot, Addis Abäba) (EMML II, 232);</bibl>
                        <bibl>EMML no. 864 (20th cent., Mänbärä Śǝllase church, ʾƎnṭoṭṭo), fols. 67r-89v (EMML III, 186).</bibl>
                     </listBibl>
```
appears as

```
            mss Bibliography
            <ul>
               
               <li></li>
               
               <li></li>
               
               <li></li>
               
               <li></li>
               
               <li></li>
               
            </ul>
```

**Comment by eu-genia** [comment]
in the title the main shelfmark should appear (now only the secondary!)

**Comment by PietroLiuzzo** [comment]
all done. please check and close issue if all is sorted.
* for wrongly encoded bibliography I have added a warning message.
* in domlib proportions are width/height and I have copied that behavior.

**Comment by eu-genia** [comment]
i think height/width is better

**Comment by eu-genia** [comment]
in layout (ex from dz-011)
```
                           <ab>
Texts II and III are not written in strict keeping with the ruled lines.</ab>
                        </layout>
```

does not show

**Comment by PietroLiuzzo** [comment]
I have added an error report for this as well.

**Comment by eu-genia** [comment]
in BLorient481: 
```
<additions>
                        <note>This volume contains copies of many gweltat, i. e. deeds of gift, grants, or donations of various kings. </note>
```
the note does not appear

**Comment by eu-genia** [comment]
"Other identiefiers:" - typo

better to move the shelfmark + other identifiers etc. to the top of the description (and not as now under Description of content)

**Comment by eu-genia** [comment]
nested msitems should be viewed hierarchically
e.g. in BLorient481

Fols. 5r-92r Octateuch

> Fols. 5r-20v Genesis
> Fols. 21r-34v Exodusbook
> Fols. 35r-45v Leviticus
> Fols. 46r-60v Numbers, Book of
> Fols. 61r-73v Deuteronomy
> Fols. 74r-82v Book ofJoshua
> Fols. 83r-90v Book ofJudges
> Fols. 91r-92a Ruth, Book of
> Colophon:
> Fol. 92r
> ተፈጸመ ፡ በዝየ ፡ ኦሪት ፡ ሰመንቱ ፡ ብሔር ፡ በሰላመ ፡ እግዚአ ብሔር ።
> Translation : ????

Fols. 94r-154v Gospels

> Fols. 94r-94v Epistle of Eusebius to Carpianus
> Fols. 95r-98v Canons of Eusebios,Canon tables
> Fols. 112r-125v Gospel of Matthew
> Fols. 126r-133v Mark, Gospel of
> Fols. 134r-145v Gospel ofLuke
> Fols. 146r-154v Gospel ofJohn

etc

(it should also be clear that incipits, colophons etc. belong to the item they belong to)

**Comment by eu-genia** [comment]
what is coded as
```
                     <origin>
                        <origPlace><placeName corresp="INS0101MadhaneAlam"/>
                           </origPlace>
```
is visualized as "current location" - wrong should be original location

**Comment by eu-genia** [comment]
also
for BLorient481
the visualization says
"Current Location: No record for Institution INS0101MadhaneAlam"
but there is a record...

**Comment by eu-genia** [comment]
```
                              <dimensions type="outer" unit="mm">
                                 <height>360</height>
                                 <width>350</width>
                              </dimensions>
```

is visualized as
H: 360 x W: 350 x D cm

the unit is defined as mm in this case

besides, we always should standardize in mm (also preferably convert domlib)

**Comment by eu-genia** [comment]
the preview says 
"ruling

THIS ab element IS WRONGLY ENCODED. Please check the schema error report to fix it."
but i have nothing on ruling in xml

**Comment by eu-genia** [comment]
> Adminstrative position
> 
> This settlement is part of Tǝgray, in Ethiopia

Administrative - typo
Omit "This settlement is" - better Region: Tǝgray; Country: Ethiopia (as "This settlement is" does not apply to rivers, mountains, regions, etc.)

**Comment by PietroLiuzzo** [comment]
ok, no prob.

Please note the improved guidelines on settlment.

https://github.com/SChAth/ScAthiop/wiki/guidelinesPlace

namely I would not replace text with elements but mark up the text which is
already there if there is any, limiting the element settlement at one
occurrence for each observation. let me know if this is ok for you.

On 10 May 2016 at 19:39, eu-genia <notifications@github.com> wrote:

> Adminstrative position
>
> This settlement is part of Tǝgray, in Ethiopia
>
> Administrative - typo
> Omit "This settlement is" - better Region: Tǝgray; Country: Ethiopia (as
> "This settlement is" does not apply to rivers, mountains, regions, etc.)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-218233601>
>



-- 
Pietro Maria Liuzzo
cel (EN): (+44)7529495584
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by PietroLiuzzo** [comment]
* ruling, pricking and wrong `ab`s now appear in separate list only if they are actually in the file
* unit for dimension is now taken from the unit. we should enforce the use of mm in the guidelines and schema, but we want to avoid that the unit is left as a standard value and the content is filled although in another unit.
* a `note` directly into `additions` was not foreseen. it is now visualized before the list
* all other issues fixed/changed

further improvements
* I have added a bit of bootstrap to collapse the main items in each msPart/msDesc
* similar manuscript now show anyway the content of the link, for checking purposes.
* also for places I have added notes referring to translations based on `@xml:lang`
* added support for `signatures` in collation
* hacked position of collation table to align more with content

**Comment by eu-genia** [comment]
the content of ab rubrication does not appear

**Comment by PietroLiuzzo** [comment]
fixed

**Comment by DenisNosnitsin1970** [comment]
we see in the visualization
endbands

No
headbands

No
tailbands

No

even when these values are absent in xml, but absent does not mean no it means we dont know, better only show when filled otherwise not show or at least not say no

**Comment by DenisNosnitsin1970** [comment]
when the code says `<textLang mainLang="gez"/>` the language name does not appear

**Comment by MassimoVilla** [comment]
While transforming my descriptions I found that:
1. as already pointed out by Denis, `Endbands NO Headbands NO Tailbands NO`  is misleading since, if the cataloguer says nothing, we actually do not know. I tried to give "Not known" as a content and transform the xml file again, but it appears immediately after Physical Description as `Physical description Not known Not known`.
2. The same thing is also valid for `crux

No

coronis

No

chirho

No`

**Comment by MassimoVilla** [comment]
Transformation of file BAVet1:
Under collation I have a "problematic" quire with a stub. I added this information within `<note/>`:
`<item xml:id="q27" n="26">
                      <dim unit="leaf">3</dim>
                      <locus from="203r" to="205v"/>
                      <note>s.l.: 3, stub before 1</note></item>`
But this does not appear in the table which is generated after the trasformation. Also, I do not see the alternative numbering I gave with @n (in the example above n=26, which differs - as you remember - from total number of quire because the initial quire A is not counted)

**Comment by MassimoVilla** [comment]
Other things which do not appear in the transformation of file BAVet1:

1. @n referring to a given line is not shown. XML file: `<note>Canons of Gangra, 20 in number: the text breaks off after Canon 18, on <locus target="#169vb" n="13"/></note>`, transformation: `the text breaks off after Canon 18, on Fol. 169vb, l.`

2. While describing item xml:id="e9" I inserted within brackets two ptr corresp to other MSS. Perhaps it is not a valid solution. Anyway, they are not shown:
`(i.e, <ptr corresp="BMLor148"/>)` appears as `(i.e, )` and
`(i.e, <ptr corresp="BAVborget2"/>)` appears as `(i.e, )`.

**Comment by DenisNosnitsin1970** [comment]
the scheme allows to have decoNote inside msItem, which is good, but they are not then re-"counted" under Decoration. can Decoration additionally show the decoNotes in msItems?

**Comment by PietroLiuzzo** [comment]
@MassimoVilla 
* please, leave unmarked text out of `note` this will be displayed in the last line of the table.
* to refer to other manuscripts, please [use `ref`](https://github.com/SChAth/ScAthiop/wiki/guidelines#references) and not `ptr[@corresp]` e.g.:
     <ref type="mss" corresp="BMLor148"/> 
If you are entering such id, please add it in the Google table so that who will do this record will assign this id to it.

@DenisNosnitsin1970 
It can be done, but what about listing it with the other decoNotes in decoDesc with a ref to the item?

**Comment by PietroLiuzzo** [comment]
now decoDesc in msItem are displayed in list at the end. Please do not repeat the information there.

**Comment by MassimoVilla** [comment]
`If you are entering such id, please add it in the Google table so that who will do this record will assign this id to it`.
I took those ids from the listWit already prepared by @eu-genia in LIT2317Senodo. I will add all of those MSS ids in the drive.

**Comment by PietroLiuzzo** [comment]
great, if they are there already, no need to add of course!

On 12 May 2016 at 16:19, MassimoVilla <notifications@github.com> wrote:

> If you are entering such id, please add it in the Google table so that who
> will do this record will assign this id to it.
> I took those ids from the listWit already prepared by @eu-genia
> <https://github.com/eu-genia> in LIT2317Senodo. I will add all of those
> MSS ids in the drive.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-218771771>
>



-- 
Pietro Maria Liuzzo
cel (DE): +49 (0) 176 61 000 606
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by sologebre** [comment]
```
               <floruit when="1563" evidence="internal"> she married to King <persName corresp="PRS7102Minas">Minas</persName> in <date>1552</date> and became the royal queen when he was entroned in <date>1559</date></floruit>
               <death when="1597" evidence="internal"> she died during the reign of her son </death>
```

the dates from floruit and death are not visualized

**Comment by MassimoVilla** [comment]
In BL Add 16,188 bibl nested within `<item xml:id="a4"/>` is not shown.

**Comment by PietroLiuzzo** [comment]
done

**Comment by MassimoVilla** [comment]
BL Add 16,188 and BL Add 11,621: information on persons (scribes, owners) does not appear.
`<listPerson><person corresp="PRS10767NewayaS" role="owner"/></listPerson>`

**Comment by PietroLiuzzo** [comment]
please, fix code as in guidelines and test again
https://github.com/SChAth/ScAthiop/wiki/guidelines#people

On 13 May 2016 at 11:39, MassimoVilla <notifications@github.com> wrote:

> BL Add 16,188 and BL Add 11,621: information on persons (scribes, owners)
> does not appear.
> <listPerson><person corresp="PRS10767NewayaS" role="owner"/></listPerson>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-218997744>
>



-- 
Pietro Maria Liuzzo
cel (DE): +49 (0) 176 61 000 606
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by MassimoVilla** [comment]
Already nested within `<particDesc/>`, just before Keywords. From BL Add 16,188:
`<particDesc>
                <listPerson>
                    <person corresp="PRS10769KesadaQ" role="owner"/> 
                    <person corresp="PRS10768Tabor" role="scribe"/>
                    <person corresp="PRS10772MalkaS" role="scribe"/> 
                </listPerson>
            </particDesc>`
It is not shown.

**Comment by PietroLiuzzo** [comment]
I think the difference is in that you need a pers name with those
attributed within a person instead of a person element only.
Il 13/Mag/2016 14:07, "MassimoVilla" <notifications@github.com> ha scritto:

**Comment by MassimoVilla** [comment]
you are right! Now I see it

**Comment by PietroLiuzzo** [comment]
let me know if then it works as expected ;)

On 13 May 2016 at 14:35, MassimoVilla <notifications@github.com> wrote:

> you are right! Now I see it
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-219044418>
>



-- 
Pietro Maria Liuzzo
cel (DE): +49 (0) 176 61 000 606
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by MassimoVilla** [comment]
it works (obviously in red since no record exists for those ids)

**Comment by MassimoVilla** [comment]
A given scribe wrote both BAVet1 and item ms_i2 in BAVet2 (so claim the cataloguers). So, in the latter he is `xml:id="h2"`. I tried to give this information within handNote as follows: `<desc>It is the same hand as <ref type="hand" target="#h1"/> of <ref type="mss" target="#BAVet1"/></desc>`. It is not shown in the preview. Is there a more appropriate way to mark up this info?

**Comment by eu-genia** [comment]
I think one should use a derived Id, something like  `xml:id="h2" corresp="BAVet1#h1"` (unless we want to create an ID for the anonymous scribe)

**Comment by MassimoVilla** [comment]
Is not `corresp` meant for pointing to the item in which that hand is used? I used it as:
`<handNote xml:id="h2" script="Ethiopic" corresp="#ms_i2">`, and two consecutive `corresp`s are not allowed.
I am trying with a relation: `<relation name="syriaca:possibly-identical" active="#h2" passive="BAVet1#h1"/>`.
Anyway, not even the item in which the hand is used `corresp="#ms_i2"` is shown in the preview.

**Comment by DenisNosnitsin1970** [comment]
1) Bibliography specific to msItem is not shown
2) DecoNote appears twice: 
in IVvostochn612 I have
```
            <decoDesc>
              <decoNote xml:id="d1" type="miniature">
                <locus target="#124v"/>
                <desc>St John the Evangelist</desc>
              </decoNote>
            </decoDesc>
```
It gets visualized as
```
Miniature notes

    miniature: Fol. 124v St John the Evangelist

Other decorations

    miniature: Fol. 124v St John the Evangelist
```

**Comment by DenisNosnitsin1970** [comment]
The extent is 164 leaves of which 1 is blank
In XML I have
                  ```
<measure unit="leaf">164</measure>
                  <measure unit="leaf" type="blank">1 (the last) <locus target="#164"/></measure>
```
The visualization is
```
extent

164 (leaf) 1 (the last) (leaf) 
```

**Comment by DenisNosnitsin1970** [comment]
I have tried to add a bibliography to DecoNote (a miniature has been published),
`
              <decoNote xml:id="d1" type="miniature">
                <locus target="#124v"/>
                <desc>St John the Evangelist</desc>
                <bibl>
                  <ptr target="bm:Vasilieva2007Christian"/>
                  <citedRange unit="page">32 fig. 8</citedRange>
                </bibl>
              </decoNote>
`
The XML seems valid but the file preview does not work with this

**Comment by eu-genia** [comment]
to Massimo's question on hands - maybe after all it is also OK as you originally suggested but simply with the proper derived ID `<desc>It is the same hand as <ref type="hand" target="BAVet1#h1"/></desc>`

However, I noted that in the preview the ref is not visualized properly and even if there is text inside this is not visible as active link
(I had e.g. `<desc>Hand of <ref target="#a1">additio 1</ref>, during the reign of <persName corresp="PRS5617IyasuII">Iyāsu II</persName> ...</desc>` and I do not see "additio 1" as a link

**Comment by PietroLiuzzo** [comment]
- [x] the rendering of desc in handNote is now sorted. 
- [ ] I agree with Eugenia for the hand pointer, although it should use @corresp 

    <desc>It is the same hand as <ref type="hand" corresp="BAVet1#h1"/></desc>

This example also shows well why it is not advisable to extensively use empty elements. in this case I would write for example 

     <desc>It is the same hand as <ref type="hand" corresp="BAVet1#h1">hand 1</ref> <ref type="mss" corresp="BAVet1"/></desc>

Although is nice I would  limit the use of listBibl[@type='relations'] to one instance for each record in which all the relevant relations are listed, focusing for this information to relations which cross the boundaries of the current record. 

- [ ] Bibliography specific to msItem
- [ ] DecoNote appears twice
- [ ] bibliography to DecoNote

**Comment by eu-genia** [comment]
(zotero -  the item was added but for some reason the tag did not get saved now fixed)
Under Miniatures now it shows

> Miniature notes
> 
>     1. miniature: Fol. 124v St John the Evangelist
>     2. Vasilieva 2007, page 32 fig. 8

the bibliography should not be numbered, possibly better in brackets ()

(and by the way is `<citedRange unit="page">32 fig. 8</citedRange>` correct or should there be an extra range for the fig.? did not find anything fitting)

**Comment by MassimoVilla** [comment]
> in this case I would write for example: `<desc>It is the same hand as <ref type="hand" corresp="BAVet1#h1">hand 1</ref> <ref type="mss" corresp="BAVet1"/></desc>` <

In the preview it appears as: It is the same hand as hand 1. No link or reference at all is given to BAVet1.

**Comment by eu-genia** [comment]
on extent:

Pietro's suggestion gets shown as

>  The extent is 164 (leaf) leaves of which 1 (leaf) (the last) Fol. 164 is blank 

I am not sure this is the vest way

I would possibly rather even formalize more by putting the value in the quantity class 
`<measure unit="leaf" quantity="164"/>`  
adding text only when necessary 
`<measure unit="leaf" type="blank" quantity="1">the last</measure>`

and in the visualization have

> Extent
> leaves (if we can use plurals for the units in the preview): 164
> of them blank: 1 (the last)

if there is a way to specify the locus for blank leaves of course)

**Comment by eu-genia** [comment]
the abbreviation for folium/folia should be f./ff. (not Fol.)

**Comment by MassimoVilla** [comment]
I think we should indicate the exact locus of the blank leaves and avoid expressions such as "the last".
"Fol." and "fols." are currently used in the ES mss records. We have to remember to batch them.

**Comment by PietroLiuzzo** [comment]
- [ ] bibliography
in file I see
````
<decoDesc>
              <decoNote xml:id="d1" type="miniature">
                <locus target="#124v"/>
                <desc>St John the Evangelist</desc>
                <bibl>
                  <ptr target="bm:Vasilieva2007Christian"/>
                  <citedRange unit="page">32 fig. 8</citedRange>
                </bibl>
              </decoNote>
            </decoDesc>
````
this should be
````
<decoDesc>
              <decoNote xml:id="d1" type="miniature">
                <locus target="#124v"/>
                <desc>St John the Evangelist</desc>
<listBibl type="secondary">                
<bibl>
                  <ptr target="bm:Vasylieva2007Christian"/> <!--or the one correct between the two-->
                  <citedRange unit="page">32 fig. 8</citedRange>
                </bibl>
</listBibl>
              </decoNote>
            </decoDesc>
````
- [ ] mss ref
I cannot reproduce the issue. with updated preview I get "It is the same hand as hand 1 (hand) of Four Gospels, Book of Hours, Missal, Prayers, Sinodos (mss) ". with link on hand 1 and the name of the manuscript. is this not the desired output?
- [ ] leafs
I would not formalize more as it feels a lot more like filling a database than marking up a text, but we can find a good way to do that for data entry. 


        <measure unit="leaf" type="blank">1 (the last) <locus target="#164"/></measure>


   and similar examples IMHO are wrong, as they are claiming that also locus and or the remaining string    are part of the measure.

        <measure unit="leaf" type="blank">1</measure> the last <locus target="#164"/>

instead gives marked up information on the extent of the mss
I will work on rendering the plural if that is necessary also for the preview. For now I have only added also a render of the type 
- [ ] f./ff.
ok, the preview will print this if there is nothing in the locus element. As @MassimoVilla says, where fols. and fol. are used in ES, they will remain as such. they can be all changed, if necessary.

**Comment by MassimoVilla** [comment]
> mss ref I cannot reproduce the issue. with updated preview I get "It is the same hand as hand 1 (hand) of Four Gospels, Book of Hours, Missal, Prayers, Sinodos (mss) ". with link on hand 1 and the name of the manuscript. is this not the desired output?

I still get: "It is the same hand as hand 1 of ." with no link. I updated the schema, refreshed OXygen. What am I forgetting?
Anyway,  I personally would find it handy to read the "idno" of the MS more than the "name": the idno is the way they are currently referred to. Not to mention special cases in which we have composite mss or simply multiple scribes, and giving all the titles as they appear in the name would be misleading. What do you ( @eu-genia , @DenisNosnitsin1970 ) think?

**Comment by eu-genia** [comment]
we should always use f./ff.
as the name of a ms we always should use the shelfmark

**Comment by eu-genia** [comment]
yes just as i have just written - i would always use the idno/shelfmark as the name of a ms; also in the preview; what we now have as "title" is a list of texts that eventually in the ms preview can be visualized in brackets under the shelfmark or in smaller font.
as for preview - i see what Pietro says i should see in BAVet2

**Comment by PietroLiuzzo** [comment]
It is the same hand as hand 1 (hand) of Vat. Aeth. 1 (mss) .

**Comment by PietroLiuzzo** [comment]
#78

**Comment by eu-genia** [comment]
it would be nice to visualize the Summary which shall be, unless provided as `<summary>`, generated from the titles of the msItems (at least the first-level ones so that we have a list of all main texts)

**Comment by eu-genia** [comment]
i have processed (incompletely) the composite ms ESmr001, it validates without errors, but i cannot preview, the error message says 

> "
> Severity: fatal
> Description: A sequence of more than one item is not allowed as the first operand of 'div'
> Start location: 139:0
> "

**Comment by eu-genia** [comment]
as for f./ff.: ideally - in manuscripts where the `<measure unit="leaf"/>`  we should only write the locus, and the visualization should be either f. (one locus) or ff. (more than one), we should not need to write abbreviations as text

of course in mss where `<measure unit="page"/>` the locus corresponds to p./pp., respectively

**Comment by PietroLiuzzo** [comment]
pp./p. done

On 30 May 2016 at 20:04, eu-genia <notifications@github.com> wrote:

> as for f./ff.: ideally - in manuscripts where the <measure unit="leaf"/>
> we should only write the locus, and the visualization should be either f.
> (one locus) or ff. (more than one), we should not need to write
> abbreviations as text
>
> of course in mss where <measure unit="page"/> the locus corresponds to
> p./pp., respectively
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-222535463>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAfAbOl2K1KH6WY4TlJECBJsGa2pF_a7ks5qGyaQgaJpZM4IaFZC>
> .
>



-- 
Pietro Maria Liuzzo
cel (DE): +49 (0) 176 61 000 606
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by PietroLiuzzo** [comment]
I have made a major reorganization of the preview to handle composite manuscript in a better way. summary is now shown in a toggle set of tabs, so that on content checking editors can check that the summary extracted from the file structure corresponds to the summary given as text (especially for EthioSpare.

**Comment by PietroLiuzzo** [comment]
these changes have now been committed to SchAthiop/preview. Please test as you go and watch out for errors, as it is very much possible that some occurred.

**Comment by eu-genia** [comment]
"extracted summary" - the sorting/grouping is funny

**Comment by eu-genia** [comment]
some things that are given once for the entire ms (e.g. binding decoNote) appear under both p1 and p2 and should be only once above (where they don't appear)

for Ethio-SPaRe mss the people are
SA: Dr Stéphane Ancel
SD: Sophia Dege
VP: Dr Vitagrazia Pisani
IF: Iosif Fridman
SH: Susanne Hummel

**Comment by eu-genia** [comment]
for "persons"
```
               <occupation type="ecclesiastic">saint</occupation>
               <occupation type="literary">writer</occupation>
```
gets visualized as
`saintwriter`

**Comment by eu-genia** [comment]
if i see right many xml files of works do not have the prefix "```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
```"
could be helpful
neither they have the types

**Comment by PietroLiuzzo** [comment]
fixed

On 1 June 2016 at 18:54, eu-genia <notifications@github.com> wrote:

> some things that are given once for the entire ms (e.g. binding decoNote)
> appear under both p1 and p2 and should be only once above (where they don't
> appear)
>
> for Ethio-SPaRe mss the people are
> SA: Dr Stéphane Ancel
> SD: Sophia Dege
> VP: Dr Vitagrazia Pisani
> IF: Iosif Fridman
> SH: Susanne Hummel
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SChAth/ScAthiop/issues/67#issuecomment-223056040>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAfAbLcS5ELoujByLYhFjx8jXBEFVtCOks5qHbk9gaJpZM4IaFZC>
> .
>



-- 
Pietro Maria Liuzzo
cel (DE): +49 (0) 176 61 000 606
Skype: pietro.liuzzo (Quingentole)
https://uk.linkedin.com/in/pietroliuzzo

**Comment by thea-m** [comment]
`<handDesc>
              <handNote xml:id="h1" script="Ethiopic">

                <seg type="ink">black, red</seg>
                <persName role="scribe" corresp="PRS10036Wansleb"/>
                <date>1666</date>
                <desc/>
              </handNote>
              <handNote xml:id="h2" script="Latin">

                <seg type="ink">red</seg>
                <persName role="scribe" corresp="PRS6378Ludolf"/>
                <date>1666</date>
                <desc/>
              </handNote>             
            </handDesc>`

is visualized as

> 
>  Hand 1
> 
> Ethiopic:
> 
> Ink: black, red
> Scribe
> 
> Johann-Michael Wansleben (scribe) scribe
> Johann-Michael Wansleben (scribe) scribe 1666
> Hand 2
> 
> Latin:
> 
> Ink: red
> Scribe
> 
> Hiob Ludolf (scribe) scribe
> Hiob Ludolf (scribe) scribe 1666

**Comment by MassimoVilla** [comment]
After transforming BL Add 16,197:

1. the information regarding the dimensions of the MS, `<note>formae 4<hi rend="apices">ae</hi> mediae</note>` (as much idiosyncratic as you want..) appears twice, and the apex ae is not shown.
2. text of the three additiones, written as follows `<q xml:lang="gez">ኍልቈ፡ መጻሕፍት፡ ዘቤተ፡ ክርስቲያን። ...</q>`, is not shown.
3. the initial sentence before a1 appears as "In this manuscript there are 3 Inventory.". Can we have the plural form in such cases?
4. The information encoded as `<listBibl type="catalogue">
                                        <bibl>
                                            <ptr target="bm:Dillmann1847BM"/>
                                            <citedRange unit="page">9a-10a</citedRange>
                                        </bibl>` appears twice as: catalogue Bibliography
Dillmann 1847, page 9a-10a.

**Comment by eu-genia** [comment]
` <bibl> 
                           <ptr target="bm:Dillmann1853Octateuch"/> 
                           <citedRange unit="page">4-5</citedRange> 
                           <citedRange unit="page">164</citedRange>
                        </bibl>`

gets visualized as
"Dillmann 1853, page page 4-5 164"

**Comment by eu-genia** [comment]
> Outer dimension
> H: 195 x W: 136 x D mm.
> 
> (proportion height/width: 1.4338 )
> 
> 195 136

(the dimensions of height width are repeated twice)

**Comment by eu-genia** [comment]
`<p>              Vers. 1: ወሀሎ ፡ ብእሲ ፡ እምነ ፡ - ሴፉ ፡  - ዘስሙ ፡ - ብናሴብ ፡ ዘኤፍሬም ።
</p>               <p> Vers. 2: ወስማ ፡ ለአሐቲ ፡ </p>
                <p>Vers. 3: ወይዐርግ ፡ - ወመጽአ ፡ ኤሊ ፡ ወደቂቁ ፡ </p>`
inside of a `<note>` gets visualized as one run-on paragraph, without line breaks

**Comment by eu-genia** [comment]
two subsequent notes are also run-on should be separate paragraphs

**Comment by PietroLiuzzo** [comment]
all the above have been resolved. Please update the scAthiop repo and test.
As this issue is overly crowded and the preview is relatively stable, I am now closing this gigantic issue and I invite you please to open a separate issue for each new fix requires, labeling it preview.
